src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -66,25 +66,23 @@
 							:container="container"
 							:close-on-select="false"
 							@select="addEmoji">
-							<button type="button"
-								:disabled="disabled"
-								class="nc-button nc-button__main emoji-picker-button"
+							<Button :disabled="disabled"
 								:aria-label="t('spreed', 'Add emoji')"
+								type="tertiary"
```

Comment:
btw @marcoambrosini  this button has no accessibility feedback after typing in the message input and then pressing `tab`